libraries/ui-library/src/components/six-tag/six-tag.tsx

```diff
@@ -65,9 +95,11 @@ export class SixTag {
           'tag--clear': this.clearable,
         }}
       >
-        <span part="content" class="tag__content">
-          <slot />
-        </span>
+        <six-tooltip ref={(el) => (this.tooltipElement = el)}>
+          <span ref={(el) => (this.contentElement = el)} part="content" class="tag__content">
+            <slot ref={(el) => (this.contentSlotElement = el as HTMLSlotElement)} />
+          </span>
+        </six-tooltip>
```

Comment:
Will this render in all cases? It seems there is no conditional to check e.g. that it only renders when the text is too long, no? I just wonder if it wouldn't be weird if you have a small tag and it renders the same text you already see in a tooltip, or did I misunderstand? 🤔

Reply:
The Tooltip is only shown if the text is truncated. It is controlled by the  [`disabled` property](https://six-group.github.io/six-webcomponents/components/six-tooltip.html#properties), which is set here: https://github.com/six-group/six-webcomponents/blob/d15348e1a1ce13a6918df0e802699e8ff7aeab30/libraries/ui-library/src/components/six-tag/six-tag.tsx#L64